services/resource-usage-tracker/requirements/_test.in

```diff
@@ -10,16 +10,20 @@
 #
 --constraint _base.txt
 
+alembic # migration due to pytest_simcore.postgres_service
 asgi-lifespan
 coverage
+docker
```

Comment:
@sanderegg I took inspiration from Catalog service, otherwise I am getting:
`Error importing plugin "pytest_simcore.docker_compose": No module named 'docker'`

Reply:
:+1: